docs/main/user-guides/upgrade.md

```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
+
+Because the mayastor chart is in active development, its helm chart values options may change over time and the convenience of the '--reuse-value' flags may also introduce nil yaml values for values which are required by helm's template engine. The 3.4.0, 3.4.1 and 3.5.0 releases face this issue. These releases require additional options to perform the default upgrade to 3.7.0 or above versions.
+Sample helm upgrade command from 3.4.0 to 3.7.0:
+```
+helm upgrade openebs openebs/openebs -n openebs --version 3.7.0 --set mayastor.image.tag=$IMG_TAG --set release.version=3.7.0 --reuse-value \
+        --set mayastor.image.pullPolicy="IfNotPresent" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.agents.core.capacity.thin.poolCommitment="250%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitment="40%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitmentInitial="40%" \
+        --set mayastor.io_engine.logLevel="info"
 ```
-Next, verify if the `io-engine` pod is in `Running` state. 
-Once, the `io-engine` pod is `Running`, verify the status of the volume rebuilds. 
 
-To make the node schedulable again, execute:
+Sample helm upgrade command from 3.4.1 to 3.7.0:
 ```
-kubectl mayastor uncordon node <node-name> <label>
+helm upgrade openebs openebs/openebs -n openebs --version 3.7.0 --set mayastor.image.tag=$IMG_TAG --set release.version=3.7.0 --reuse-value \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.agents.core.capacity.thin.poolCommitment="250%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitment="40%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitmentInitial="40%" \
+        --set mayastor.io_engine.logLevel="info"
+```
+
+Sample helm upgrade command from 3.5.0 to 3.7.0:
 ```
-Ensure that there are no in-progress rebuilds.
+helm upgrade openebs openebs/openebs -n openebs --version 3.7.0 --set mayastor.image.tag=$IMG_TAG --set release.version=3.7.0 --reuse-value \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.image.repoTags.controlPlane="" \
+        --set mayastor.agents.core.capacity.thin.poolCommitment="250%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitment="40%" \
+        --set mayastor.agents.core.capacity.thin.volumeCommitmentInitial="40%"
+```
+The 3.6.0 release does not require special options to upgrade to 3.7.0 or above versions.
```

Comment:
Just add the command instead of the statement?

Reply:
Done..